src/components/pages/ClientDashboard/RenderClientDashboard.js

```diff
@@ -6,14 +6,14 @@ import { Spinner } from '../../common';
 
 const RenderClientDashboard = () => {
   const clientContext = useContext(ClientContext);
-  const { client } = clientContext;
+  const { client, accountType } = clientContext;
 
   return (
     <div>
       {Object.keys(client).length < 1 ? (
         <Spinner />
       ) : (
-        <ClientDashboardContainer client={client} />
+        <ClientDashboardContainer client={client} type={accountType} />
```

Comment:
Take a look at the ExampleList page component. This should be reversed with the ClientDashboardContainer. The render component is where most of the JSX should reside.